laser_encoders/validate_models.py

```diff
@@ -0,0 +1,114 @@
+import os
+import tempfile
+
+import pytest
+
+from laser_encoders.download_models import LaserModelDownloader
+from laser_encoders.language_list import LASER2_LANGUAGE, LASER3_LANGUAGE
+from laser_encoders.laser_tokenizer import initialize_tokenizer
+from laser_encoders.models import initialize_encoder
+
+
+@pytest.mark.parametrize("lang", LASER3_LANGUAGE)
+def test_validate_language_models_and_tokenize_laser3(lang):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        print(f"Created temporary directory for {lang}", tmp_dir)
+
+        downloader = LaserModelDownloader(model_dir=tmp_dir)
+        if lang in ["Kashmiri", "kas", "central kanuri", "knc"]:
```

Comment:
For these languages, instead of asserting a `ValueError`, can you check each of the available options (scripts).
e.g. if we have `Kashmiri` then check the downloader + encoder for those options: ["kas_Arab", "kas_Deva"]. Otherwise we don't know if they work.

On another note, cna you lowercase `Kashmiri` in the `language_list.py`

Reply:
> Otherwise we don't know if they work.

Actually, we know, because in the big language list, there already are their versions with the script specified: https://github.com/facebookresearch/LASER/blob/MLH-dev/laser_encoders/language_list.py#L268. 

So I don't think we need any more changes on this particular issue.

Reply:
ok great!

Reply:
So the correct code for Kashmiri is kashmiri?

Reply:
Yes. It's just so we're consistent. However, there's no need to check for the scripts I mentioned above as David noticed we already have those in the language list.